src/rs.cpp

```diff
@@ -170,7 +171,7 @@ rs2_context* rs2_create_context(int api_version, rs2_error** error) BEGIN_API_CA
 {
     verify_version_compatibility(api_version);
 
-    return new rs2_context{ std::make_shared<librealsense::context>(librealsense::backend_type::standard) };
+    return new rs2_context{ std::make_shared< librealsense::context >( librealsense::backend_type::standard ) };
```

Comment:
I do not like these spaces beside the "<>", but... your style :-)

Reply:
No, it's *our* style.
There is a .clang-format file that you should be using. See me tomorrow if you do not :)

Reply:
ok - feel free to send it to me